app/slashcommands/command_share.go

```diff
@@ -240,6 +240,14 @@ func (sp *ShareProvider) doInviteRemote(a *app.App, args *model.CommandArgs, mar
 		return responsef("Must specify a valid remote cluster id to invite.")
 	}
 
+	all, err := a.GetSharedChannelRemotes(args.ChannelId)
```

Comment:
Not a blocker, 1/5, but this will fetch all the columns for a given channel when the channel exists in the `SharedChannelRemotes` table, I think a more appropriate query here would be an existence check like `select 1 as one from SharedChannelRemotes where ChannelId = x`

Reply:
Agree. We will need to add a new method. Will do. :+1:

Reply:
Done.